Constructors.R

```diff
@@ -0,0 +1,34 @@
+# Used as guard
+ConstructorsFile <- T
+
+# This module is able to build `CoreVariable` objects
+# With one of the pmf or cdf functions.
+# To extract cdf from pmf it's enough to integrate (-Inf, x)
+# To extract pmf from cdf it's enough to differenciate
+
+if (!exists('CoreVarFile'))
+    source('CoreVariable.R')
+
+# Creates a CoreVariable from a pmf
+BuildFromPMF <- function(pmf) {
+  cdf <- function(x) {
+    return(integrate(pmf, -Inf, x)$value)
+    # TODO: Fix precision errors
+  }
+  
+  variable <- CoreVariable(cdf=cdf, pmf=pmf)
+  
+  return(variable)
+}
+
+# Creates a CoreVariable from a cdf
```

Comment:
aici se poate imbunatati putin tehnica. in loc sa luam intervalul (x, x + eps), putem sa luam intervalul (x - eps/2, x + eps/2)